src/analytics/analytics.js

```diff
@@ -327,6 +342,30 @@ export class Analytics {
         this.nextStepNumber += 1;
 
         this.updateCsvExportLink();
+
+        // wider tolerance for associating local cameravis patches with
+        // potentially remote region cards
+        const patchTolerance = 3000;
+        if (Math.abs(regionCard.endTime - Date.now()) > patchTolerance) {
+            return;
+        }
+
+        const desktopRenderer = realityEditor.gui.ar.desktopRenderer;
+        if (!desktopRenderer) {
+            return;
+        }
+
+        const patches = desktopRenderer.cloneCameraVisPatches();
```

Comment:
I suppose that these new methods in desktopRenderer are is some other PR. Is this set of patches limited to the time interval of this region card?

Reply:
Still could not try because of the missing changes to desktopRenderer in remote operator addon. But it seems that most of new commits here are in the other PR as well - https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/pull/481

Reply:
#481 takes the non-patch-related changes from this branch. I'm planning to revisit this patches work after that PR since I still need to solve patch deletion and synchronization. The corresponding remote operator PR is https://github.com/ptcrealitylab/vuforia-spatial-remote-operator-addon/pull/108